### ISSUE TITLE
New orders forces the market view to scroll up

### ISSUE THREAD

**Issue by robrigo** [bitshares/bitshares-ui#370]
The user is forced to the top of the "market view" when a new order is placed. This bug causes a bad UX if a user is trying to view information that requires scrolling down when new orders are being placed frequently.

Notice in the following gif how a new order flashing on the sell side of the books forces the scroll view back to the top:

![exchange-view-scrolls-up-when-order-placed](https://user-images.githubusercontent.com/590263/30173246-0d685ed0-93c5-11e7-938f-29f3fd6c1ca9.gif)

I am using Version 2.0.170830 of the BitShares2-light wallet.

**Comment by wmbutler** [comment]
I believe this bug was created when another user complained about the opposite problem when orders were presented in the pinned left view. Whoever claims this need to be sure that it works in both order views (below the market and to the left of the market). Allocating an hour since this may be tricky.

**Comment by svk31** [comment]
Unable to reproduce this, which os are you using?

**Comment by robrigo** [comment]
@svk31 I'm on macOS Sierra (latest)

**Comment by wmbutler** [comment]
Any chance you could post a screencast of it?

**Comment by robrigo** [comment]
@wmbutler I attempted to do so by creating the gif above using LICEcap. That gif is a recording of my screen scoped in on the relevant parts of the UI.

In order to reproduce this, I did the following:
1) Navigate to the bitUSD / BTS trading pair under the Exchange tab on the desktop wallet client.
2) Opened bitshares via OL on a spare laptop, separate account.
3) Scrolled down to the bottom of the scroll bar in the market view of the desktop wallet and started recording my screen.
4) Put in a small buy order for bitUSD at the top of the books on the openledger machine.
5) Observed as the screen leaped up on the desktop wallet machine (the gif above shows this behavior)

**Comment by wmbutler** [comment]
Can you post the LICEcap gif?

**Comment by robrigo** [comment]
The gif is attached inline in the O.P. @wmbutler. Can you see the market gif?

**Comment by wmbutler** [comment]
My screen is big enough that I can't reproduce this...not really sure how to proceed. What would you like to happen instead?

**Comment by robrigo** [comment]
@wmbutler I propose that moving the user's scroll position when an order is triggered be removed. It makes the lower half of the page unusable when there is a lot of market activity, because the screen keeps jumping to the top scroll position every few seconds. I'm not exactly sure what is triggering it but I will look into building a dev environment so I can dig into this bug myself later in the week.

**Comment by wmbutler** [comment]
Nobody has claimed it so I'll likely move it to the next Sprint since we are nearing the end of this one.

**Comment by wmbutler** [comment]
lol, And @calvinfroedge just took it!

**Comment by calvinfroedge** [comment]
I can handle this as I made the other change

**Comment by robrigo** [comment]
@calvinfroedge MVP

**Comment by calvinfroedge** [comment]
@robrigo Sorry man, but I am also having a hard time reproducing this issue. As I understand you have two instances of the app open at the same time and creating a buy / sell order in one of those instances causes jumpiness in the other. I am seeing the notifications, but I am not seeing the behavior you describe. If you can provide more detailed reproduction steps (perhaps show your two screens side by side) I can dig further into it.

**Comment by robrigo** [comment]
@calvinfroedge when the market is active, you don't need another computer / account to trigger it. I produced the following example in the BTS: bitUSD market.

![wonky-forced-scroll-positioning](https://user-images.githubusercontent.com/590263/30294776-43c4ff02-970c-11e7-8ae1-1fe1db001496.gif)

After a little more testing, it seems to only happen at a range of window widths that renders a specific layout. When the width of the window is small, the scroll bug doesn't occur. As I widen my window and the elements reposition, it is triggered.  When the "Market Trades" container is to the right of the "SELL BTS" container in the gif above, notice how the scroll position is continually forced up, and I need to scroll back down to see the bottom of the page.

**Comment by svk31** [comment]
Think I know how to fix this.

**Comment by wmbutler** [comment]
I have to agree. Let's table this. It's really not even a bug so much as an inconvenience.

**Comment by robrigo** [comment]
I disagree that this should be classified as a feature proposal, due to the fact that I can't interact with the bottom half of the market view on an active pair at my default window size. Users that encounter the same problem may be prevented from trading if they aren't aware they can simply resize the window.

I'm ok with the prioritization set to low, but still think that this issue should be classified as a bug.

**Comment by landry314** [comment]
This is much more than an inconvenience to me. It jumps to the top immediately whenever I scroll down making it very hard to see settlement orders or scroll down the orders list because they are slightly below the bottom of the screen. If the monitor cant display all the "tiles" in the exchange at once then the scroll down is necessary and it just keeps jumping to the top of the scroll bar, for me it is one second, not 5-10.
